msteams-platform/toolkit/publish.md

```diff
@@ -50,15 +50,15 @@ Perform the following steps to upload app package:
 
```

Comment:
L42: **Provision in the cloud** -> **Provision**
L44: Select **Zip Teams metadata package** under **DEPLOYMENT**. -> Select **Zip Teams App Package** under **UTILITY**.

Reply:
Updated.